L/libdatachannel/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libdatachannel"
+version = v"0.20.2"
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/paullouisageneau/libdatachannel.git", "9cbe6a2a1f21cde901bca9571581a96c6cda03cf"),
+    ArchiveSource("https://github.com/phracker/MacOSX-SDKs/releases/download/10.15/MacOSX10.14.sdk.tar.xz", 
+               "0f03869f72df8705b832910517b47dd5b79eb4e160512602f593ed243b28715f")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+if [[ ${target} == x86_64*mingw* ]]; then
+    export OPENSSL_ROOT_DIR="${prefix}/lib64"
+fi
+
+if [[ "${bb_full_target}" == x86_64-apple-darwin* ]]; then
+    # LLVM 15+ requires macOS SDK 10.14.
+    pushd $WORKSPACE/srcdir/MacOSX10.*.sdk
+    rm -rf /opt/${target}/${target}/sys-root/System
+    cp -ra usr/* "/opt/${target}/${target}/sys-root/usr/."
+    cp -ra System "/opt/${target}/${target}/sys-root/."
+    export MACOSX_DEPLOYMENT_TARGET=10.14
+    popd
+fi
+
+cd $WORKSPACE/srcdir/libdatachannel
+git submodule update --init --recursive --depth 1
+cmake -B build \
+    -DUSE_GNUTLS=0 \
+    -DUSE_NICE=0 \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release
+cmake --build build --parallel ${nproc}
+cmake --install build
+"""
```

Comment:
```suggestion
install_license $WORKSPACE/srcdir/libdatachannel/LICENSE
"""
```

The auditor was flagging a missing license.